src/index.js

```diff
@@ -1,4 +1,16 @@
-const root = document.getElementById('root');
-const header = document.createElement('h1');
-header.appendChild(document.createTextNode('Hello World'));
-root.appendChild(header);
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import App from './components/App';
+import rootReducer from './reducers/index';
+import './index.css';
+
+const store = createStore(rootReducer);
+
```

Comment:
- [ ] It adds example books to the initial state object [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/InitialBooks.png) (Your initial books need to be initialized here and not in your reducers as requested by the project requirements)

Reply:
Carried the "bookList" into src/index.